src/gpm-control.c

```diff
@@ -390,6 +397,35 @@ gpm_control_hibernate (GpmControl *control, GError **error)
 		}
 	}
 
+#ifdef WITH_LIBSECRET
+	/* we should perhaps lock keyrings when sleeping #375681 */
+	lock_libsecret = g_settings_get_boolean (control->priv->settings, GPM_SETTINGS_LOCK_KEYRING_SUSPEND);
+	if (lock_libsecret) {
+    libsecret_cancellable = g_cancellable_new();
```

Comment:
Indents....and other problems, same as in file before.
the whole code block has a bad code-style....